FTB/Signatures/CrashInfo.py

```diff
@@ -1148,75 +1149,88 @@ def __init__(self, stdout, stderr, configuration, crashData=None):
+                #      r8=000000e87fbfc140  r9=000000000001fffc r10=0000000000000649
+                #     r11=00000285ef25a000 r12=00007ff74d9239a0 r13=fffa7fffffffffff
+                #     r14=000000e87fbfd0e0 r15=0000000000000003
                 #     iopl=0         nv up ei pl nz na pe nc
-                #     cs=0033  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00000200
+                #     cs=0033  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00010200
                 if line.startswith("cs="):
                     inEcxrData = False
                     continue
 
-                # First extract the line, example:
-                #     rax=0000000000000000 rbx=0000000000008000 rcx=00000000000005af
+                # First extract the line
+                # 32-bit example:
+                #     eax=02efff01 ebx=016fddb8 ecx=2b2b2b2b edx=016fe490 esi=02e00310 edi=02e00310
+                # 64-bit example:
+                #     rax=00007ff74d8fee30 rbx=00000285ef400420 rcx=2b2b2b2b2b2b2b2b
                 matchLine = re.search(RegisterHelper.getRegisterPattern(), line)
-                if matchLine != None:
+                if matchLine is not None:
                     ecxrData.extend(line.split())
 
-                # Next, put the rax, rbx, rcx, etc. entries into a list of their own, then iterate
+                # Next, put the eax/rax, ebx/rbx, etc. entries into a list of their own, then iterate
                 match = re.search(cdbRegisterPattern, line)
                 for instr in ecxrData:
                     match = re.search(cdbRegisterPattern, instr)
-                    if match != None:
+                    if match is not None:
                         register = match.group(1)
                         value = long(match.group(2), 16)
                         self.registers[register] = value
 
             # Crash address
-            if line.startswith("Exception Faulting Address:"):
-                # Example:
-                #     Exception Faulting Address: 0x7fef86c13e4
-                address = line.split(": ")[1]
+            if line.startswith("ExceptionAddress:"):
+                # 32-bit example:
+                #     ExceptionAddress: 00404c59 (js_32_dm_windows_62f79d676e0e!JSObject::is+0x00000002)
+                # 64-bit example:
+                #     ExceptionAddress: 00007ff74d469ff3 (js_64_dm_windows_62f79d676e0e!JSObject::is+0x000000000000000a)
+                address = line.split()[1]
                 self.crashAddress = long(address, 16)
 
             # Crash instruction
-            if line.startswith("Faulting Instruction:"):
-                # Example:
-                #     Faulting Instruction:01206fbd int 3
-                cInstruction = line.split(":")[1].split(None, 1)[1]
+            # 64-bit binaries have a backtick in their addresses, e.g. 00007ff7`1e424e62
+            lineWithoutBacktick = line.replace("`", "")
```

Comment:
Is there only one backtick per line? If so, you might want to limit the replace to the first occurrence.

Also, do you need  the original line with the backtick? If not, just do line = line.replace... to keep the code simple.